src/kuzzle.js

```diff
@@ -1,9 +1,9 @@
 var
   uuid = require('uuid'),
-  KuzzleDataCollection = require('./kuzzleDataCollection'),
-  KuzzleSecurity = require('./security/kuzzleSecurity'),
-  KuzzleMemoryStorage = require('./kuzzleMemoryStorage'),
-  KuzzleUser = require('./security/kuzzleUser'),
+  Collection = require('./kuzzleCollection'),
```

Comment:
I don't really see why, we already renamed contructors in them. `kuzzle` in the filename does not bring anything to the comprehension, and is not aligned with constructor name.